components/trial-bar/dropdown.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import Dropdown from '~/components/menu-dropdown'; // `~` is replaced with design-system-react at runtime
+import DropdownTrigger from '~/components/menu-dropdown/button-trigger';
+import MenuDropdown from '~/components/menu-dropdown/menu-dropdown';
+import Button from '~/components/button';
+
+import { TRIAL_BAR_DROPDOWN } from '../../utilities/constants';
+
+// This component accepts the same props as MenuDropdown.
+// eslint-disable-next-line react/forbid-foreign-prop-types
+const { propTypes } = MenuDropdown;
+
+const TrialBarDropdown = (props) => {
+	const { label, ...rest } = props;
+	return (
+		<Dropdown {...rest} inverse>
```

Comment:
`slds-grid` class name seems to be missing on the dropdown. See prop `triggerClassName` on dropdown component.

Reply:
Added `triggerClassName` prop